product_docs/docs/hadoop_data_adapter/2/10c_example_order_by_pushdown.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "Example: ORDER BY pushdown "
+---
+
+This example shows ORDER BY pushdown between the foreign tables of the same remote HIVE/SPARK server to that remote HIVE/SPARK server:
```

Comment:
Use Hive/Spark
I think "that" should be "the"

Reply:
I think instead of "to that" you want "as the"

Reply:
Done changes as per both comments